integration/helpers.go

```diff
@@ -405,3 +405,14 @@ func FindAll(t testing.TB, ctx context.Context, collection *mongo.Collection) []
 
 	return FetchAll(t, ctx, cursor)
 }
+
+// generateDocuments generates documents with _id ranging from startID to endID.
+// It returns bson.A containing bson.D documents.
+func generateDocuments(startID, endID int32) bson.A {
```

Comment:
This is similar to what was done in https://github.com/FerretDB/FerretDB/pull/1769/files#diff-8307cfe62efa2f64d4ef2d0aff407a8af0986af9f9efe7f98de217e13fe89297R390